projects/packages/jetpack-mu-wpcom/src/features/blog-privacy/blog-privacy.php

```diff
@@ -18,11 +18,15 @@
 /**
  * Filters the robots.txt contents based on the value of the wpcom_data_sharing_opt_out option.
  *
- * @param string     $output The contents of robots.txt.
- * @param int|string $public The value of the blog_public option.
+ * @param string|null $output The contents of robots.txt.
+ * @param int|string  $public The value of the blog_public option.
```

Comment:
Along with the below, we might just declare what [the hook itself does](https://developer.wordpress.org/reference/hooks/robots_txt/).
```suggestion
 * @param string $output The contents of robots.txt.
 * @param bool   $public Whether the site is considered public.
```